core/src/main/java/org/openmdx/application/mof/mapping/spi/MapperTemplate.java

```diff
@@ -84,6 +86,17 @@ protected String renderAnnotation(final String annotation) {
 		return annotationRenderer.apply(annotation);
 	}
 
+	// TODO: DKAJ mdx-41
+	protected void possiblyOutputJavaLangDeprecatedAnnotation(ElementDef elementDef, boolean isClassLevelAnnotation) {
```

Comment:
```suggestion
	protected void mapDeprecatedAnnotation(ElementDef elementDef) {
```
da sich `isClassLevelAnnotation` innerhalb der Methode durch `elementDef instanceof ClassDef` ersetzen lässt